### ISSUE TITLE
ResNet56

### ISSUE THREAD

**Issue by Slawlight** [LiyuanLucasLiu/RAdam#10]
I am sorry, I come again. Can you  tell me the hyperparameters setting on ResNet56? I got a very poor test accuracy 91.1 which is worse than that in ResNet20 .  I set lr=0.01 and weight-decay=1e-4. Is there something wrong?

**Comment by LiyuanLucasLiu** [comment]
Sorry, we only conduct experiments with the default architecture (in the original repo), and haven't tried ResNet56. As to other architectures, there are some recipes for SGD [here](https://github.com/bearpaw/pytorch-classification/blob/master/TRAINING.md). In our experiments, we adopt most of hyper parameters (which is originally tuned for SGD), and tune the learning rate for Adam / RAdam. 

My suggestion is you can first check whether your current setting works for SGD.   

Thanks and good luck!

**Comment by Slawlight** [comment]
I have tried again. Radam can also work on Resnet56. The hyperparameters setting is the same as that used on ResNet20. 
Very nice work.

**Comment by LiyuanLucasLiu** [comment]
Cool! Thanks for letting us know : -)

**Comment by Slawlight** [comment]
This is the result.
<img width="623" alt="屏幕快照 2019-08-21 上午10 51 10" src="https://user-images.githubusercontent.com/18410535/63398936-aabc7680-c401-11e9-8f7b-fe5506f47c78.png">
<img width="610" alt="屏幕快照 2019-08-21 上午10 51 04" src="https://user-images.githubusercontent.com/18410535/63398946-b3ad4800-c401-11e9-8a83-7a902006e5c4.png">
